src/context/alert.tsx

```diff
@@ -85,7 +85,7 @@ const errorNameToTranslationKeyMappings: Record<string, string> = {
   LoginError: "systemErrorLoginIssue",
   AuthTakenError: "alreadyUsedQRCode",
   AuthExpiredError: "expiredQR",
-  AuthNotFoundError: "expiredQR",
+  AuthNotFoundError: "invalidInputQR",
```

Comment:
is this error message aligned with Eida's understanding? noticed it wasn't stated in the story, might be good to check in with her to confirm?

Reply:
Eida suggested [wrong format](https://www.notion.so/sally-wallet/Error-Message-4d003233e7364aebab959ae8a2e97bbe) error message. Have updated the code.